cmd/aws/main.go

```diff
@@ -22,7 +22,9 @@ func main() {
 	log.SetFormatter(helpers.LogFormatter)
 	log.Infof("starting api: version=%s buildTime=%s sha1ver=%s", version, buildTime, sha1ver)
 
-	conf, err := config.LoadConfig[config.IotAWS]()
+	conf, err := config.LoadConfig[config.IotAWS](map[string]interface{}{
```

Comment:
Can we do something a bit more elegant and generic here? 
I mean, define the defaults as an instance of the config type
```
var DefaultIotAWS = IotAWS{
	AWSBidirectionalQueueName: "Lamassu-IoT-SYNC-EventBridgeOutput6A8BBEEC-LaYbNuW753SC",
}
```
Then provide this instance as default, and modify the LoadConfig method to transform the struct to a map as expected by vipper using the mapstructure support

Reply:
Sounds good, I'll check these days. Thanks!